Modules/QR Reader/MITScannerMgr.m

```diff
@@ -86,7 +88,9 @@ - (void)setupCaptureSession
     self.deviceInput = [[AVCaptureDeviceInput alloc] initWithDevice:self.captureDevice error:&error];
     if( self.deviceInput == nil )
```

Comment:
@yevgmotov You are correct there; I mistook the deviceInput for the captureDevice. Apologies for the mixup!